components/server/src/installation-admin/installation-admin-controller.ts

```diff
@@ -6,17 +6,25 @@
 
 import { injectable, inject } from 'inversify';
 import * as express from 'express';
-import { InstallationAdminDB } from '@gitpod/gitpod-db/lib';
+import { InstallationAdminDB, UserDB, WorkspaceDB } from '@gitpod/gitpod-db/lib';
+import { Data } from '@gitpod/gitpod-protocol'
 
 @injectable()
 export class InstallationAdminController {
     @inject(InstallationAdminDB) protected readonly installationAdminDb: InstallationAdminDB;
+    @inject(UserDB) protected readonly userDb: UserDB
+    @inject(WorkspaceDB) protected readonly workspaceDb: WorkspaceDB
 
     public create(): express.Application {
         const app = express();
 
         app.get('/data', async (req: express.Request, res: express.Response) => {
-            const data = await this.installationAdminDb.getData();
+            const data: Data = {
+                installationAdmin: await this.installationAdminDb.getData(),
+                totalUsers: await this.userDb.getUserCount(false),
```

Comment:
NIT: I personally would exclude the buildin user here. But that's maybe more a matter of taste. If you get a change request from the team web app review, you could consider changing this. Otherwise, I'm fine with just merging it as it.
```suggestion
                totalUsers: await this.userDb.getUserCount(true),
```

Reply:
@corneliusludmann What's a built-in user? 🙏🏽 I tested it and despite only having one user in the preview env, I got `"totalUsers":2`.

Reply:
We always add a user for “workspace probes”.

https://github.com/gitpod-io/gitpod/blob/240197f239faf9a5b1fb67c5260f3564637b5e52/components/gitpod-db/src/user-db.ts#L122

https://github.com/gitpod-io/gitpod/blob/7abe78c91792c18b0c22103c890b25ea3a4cdf05/components/gitpod-db/src/typeorm/migration/1592203031938-Baseline.ts#L84

Workspace probes are started here:

https://github.com/gitpod-io/gitpod/blob/1c0ee09cdc6a827c5e4258c1c8369eb8e6914c5f/components/server/ee/src/workspace/workspace-health-monitoring.ts#L36

This is a regular test, that the cluster is healthy and we can start workspaces.

Reply:
Thanks! In this case I would also exclude built-in users, as I believe it doesn't serve your purpose @Pothulapati. Once you have this change I'll give my thumbs up.

Reply:
Done! 👍🏼

Reply:
Interestingly, I still see the count to start from`2`. Is this because of the presence of `admin` by default? 🤔

Reply:
Just verified this! and It's actual users without admin! 2 of my accounts (I logged in for testing) and 1 of @laushinka! Checks out with the current count of `3` without any admin/built-in users.

This is fixed.

Reply:
Yess, you can check the actual users here https://tarun-telemetry-users.staging.gitpod-dev.com/admin/users 🤗